### PR TITLE
Feat/#64 cleanup

### PR DESCRIPTION
**Pull request by ShravanSunder**

### CODE REVIEW

File: .husky/pre-commit [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+yarn lint-staged -p 2
```
Comment (ssp6) [inline .husky/pre-commit:4]: lint-staged is cool. Just implemented this on another project and really helps to find issues before getting to PR stage

File: packages/hardhat-ts/.prettierrc [hunk 2]
```diff
@@ -1,10 +1,27 @@
...
-   "endOfLine": "lf"
+  "singleQuote": true,
+  "trailingComma": "es5",
+  "requirePragma": false,
+  "arrowParens": "always",
+  "bracketSameLine": true,
+  "tabWidth": 2,
+  "printWidth": 160,
+  "endOfLine": "lf",
+  "overrides": [
```
Comment (ssp6) [inline packages/hardhat-ts/.prettierrc:10]: Nice, didn't know prettier could be applied to hardhat

File: packages/hardhat-ts/package.json [hunk 3]
```diff
@@ -1,63 +1,87 @@
...
+    "format:check": "yarn prettier -c ./deploy/**/*.ts ./scripts/**/*.ts ./test/**/*.ts ./helpers/**/*.ts ./contracts/**/*.sol "
+  },
+  "dependencies": {
+    "@atixlabs/hardhat-time-n-mine": "^0.0.5",
+    "@defi-wonderland/smock": "^2.0.7",
+    "@eth-optimism/hardhat-ovm": "^0.2.4",
+    "@ethereum-waffle/ens": "^3.3.1",
+    "@ethereumjs/vm": "^5.6.0",
+    "@ethersproject/abi": "^5.5.0",
+    "@ethersproject/abstract-provider": "^5.5.1",
```
Comment (ssp6) [inline packages/hardhat-ts/package.json:31]: This doesn't appear to be used anywhere, a usage of it was actually removed from `packages/vite-app-ts/src/functions/getNetworkInfo.ts`

File: packages/vite-app-ts/package.json [hunk 4]
```diff
@@ -22,29 +25,31 @@
     "@babel/runtime": "^7.16.7",
     "@emotion/react": "^11.7.1",
     "@ethersproject/abi": "^5.5.0",
+    "@ethersproject/abstract-provider": "^5.5.1",
```
Comment (ssp6) [inline packages/vite-app-ts/package.json:28]: Same as above, can't find use case (possibly just misunderstanding of how it works)

File: packages/vite-app-ts/.env [hunk 5]
```diff
@@ -0,0 +1,14 @@
+VITE_APP_TARGET_NETWORK=localhost
+HARDHAT_TARGET_NETWORK=localhost
+
+# network
```
Comment (ssp6) [inline packages/vite-app-ts/.env:4]: Just double-checking, is all of this safe to be here?

**Review by ssp6 — COMMENTED** [review]
There's a lot going on here but it's much prettier which is always nicer to work with.

Left a few comments, they're likely okay but just stood out. 

Left a comment in telegram about the husky not working on my machine. May just be my machine but it really is a blocker for using the project if it isn't.

File: packages/vite-app-ts/.env (hunk 5 above)
Comment (ShravanSunder) [reply packages/vite-app-ts/.env:4]: yeah, its all stuff austin pays for and is ok with other people working with it.  It was all hardcoded before in js files.

File: packages/vite-app-ts/package.json (hunk 4 above)
Comment (ShravanSunder) [reply packages/vite-app-ts/package.json:28]: i forgot the reason why i needed it.  but it doesn't hurt as vite will tree shake uneeded things

**Comment by grothem** [comment]
Prettier and linting hooks are working. I deployed contracts and ran, everything looks good.

**Review by grothem — APPROVED** [review]

**Pull request merged**